mlserver/repository.py

```diff
@@ -29,8 +29,11 @@ async def list(self) -> List[ModelSettings]:
             matches = glob.glob(pattern, recursive=True)
 
             for model_settings_path in matches:
-                model_settings = self._load_model_settings(model_settings_path)
-                all_model_settings.append(model_settings)
+                try:
+                    model_settings = self._load_model_settings(model_settings_path)
+                    all_model_settings.append(model_settings)
+                except:
```

Comment:
should we be specific with the exception here i.e. `FileNotFoundError`?

Reply:
I think we need to be as wide as possible as otherwise what happens if someone includes a broken model settings file. This would stop all models from loading? 
If there is a set of standard PyDantic exceptions we could catch maybe?